opentelemetry-sdk/setup.py

```diff
@@ -56,4 +56,14 @@
         "/tree/master/opentelemetry-sdk"
     ),
     zip_safe=False,
+    entry_points={
+        "opentelemetry_context": [
+            "contextvars_context = "
+            "opentelemetry.context.contextvars_context:"
+            "ContextVarsContext",
```

Comment:
I think this entry point should be in the API setup, since `ContextVarsContext` is defined in the API.

Reply:
yes, i moved all the entrypoints in the api for now